org.eclipse.xtext.xtext.generator/src/org/eclipse/xtext/xtext/generator/ui/codemining/CodeMiningFragment.xtend

```diff
@@ -39,11 +39,11 @@ class CodeMiningFragment extends AbstractStubGeneratingFragment {
 			projectConfig.eclipsePlugin.manifest.importedPackages += "org.eclipse.xtext.ui.codemining;resolution:=optional"
 		}
 		new GuiceModuleAccess.BindingFactory => [
-			addConfiguredBinding("CodeMinding", '''
-				binder.bind(�'org.eclipse.jface.text.codemining.ICodeMiningProvider'.typeRef�.class)
-					.to(�codeMiningProviderClass�.class);
-				binder.bind(�'org.eclipse.xtext.ui.editor.reconciler.IReconcileStrategyFactory'.typeRef�.class).annotatedWith(�Names.typeRef�.named("codeMinding"))
```

Comment:
hmmm wont this break clients? did you check if this is used somewhere

Reply:
@szarnekow wdyt? break and write something in release notes?

Reply:
By doing a simple text search, It's only referenced in `AbstractDomainmodelUiModule#configureCodeMinding` and `AbstractArithmeticsUiModule#configureCodeMinding` which don't seem to be used

Reply:
If we consider only this generated binding in `src-gen` a regeneration will fix things in the generated code.
If one had customized such binding, they should get an error if they had used `@Override`.
In general, I'd be in favor of this change, of course, documenting it.

Reply:
@LorenzoBettini i meant on user side. `@Inject @Named("...")`

Reply:
@cdietrich you're right!
However, it would be great if we could get rid of the typo ;)

Reply:
yes i assume nobody is using it. at least in xtext codebase we seem not to